packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -162,14 +162,16 @@ const Column = styled.div`
   }
 
   @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
-    margin: 0px;
-    width: 100%;
+    width: 0;
```

Comment:
I think using `display: none` is fine here.

Reply:
Yeah, I changed it !